src/tests/suite/daos_degrade_ec.c

```diff
@@ -591,9 +594,27 @@ degrade_ec_update(void **state)
 		recx.rx_nr = TEST_EC_STRIPE_SIZE;
 		recx.rx_idx = i * TEST_EC_STRIPE_SIZE;
 		memset(data, 'a' + i, TEST_EC_STRIPE_SIZE);
-		insert_recxs("d_key", "a_key", 1, DAOS_TX_NONE, &recx, 1,
+		inset_recxs_dkey_uint64(&dkey_int, "a_key", 1, DAOS_TX_NONE, &recx, 1,
```

Comment:
is the bug related to integer keys?   Are we removing some coverage here by changing the test to use them instead of standard keys?

Reply:
the bug is not related to integer dkey, why I change the test to use integer dkey is to reproduce the assertion, need to make sure the test case in below L602 ~ L615 will generate update request to the 2nd obj group. 
Actually it is not must to use integer dkey, can select some special string dkey to make it happened to be hashed to 2nd group as well. Just that I changed to use integer dkey for that purpose at beginning when I make the patch so did not change it back later.

Reply:
Nah, it's fine to use integer key.   I was mostly just wondering if we lose any coverage with the change.